test/e2e/e2e-test/src/test/resources/expected/metricsservice/services.yml

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+services:
+  - key: not null
+    label: istio-dp::ratings.default
+  - key: not null
+    label: istio-dp::reviews.default
+  - key: not null
+    label: istio-dp::productpage.default
+  - key: not null
+    label: istio-dp::details.default
+  - key: not null
+    label: istio-dp::istio-ingressgateway
```

Comment:
Such as this one should be same as ALS did. https://github.com/apache/skywalking/blob/1b1f08511ccb8542d96f29dcaee72c73492b74f9/test/e2e/e2e-test/src/test/resources/expected/als/services.yml#L16-L26

More importantly, the instance name should be pod's name.

Reply:
This is `services.yaml`

Reply:
I know. I just take an example, because we don't verify the instance name in the test. The point is, we need ALS and MetricsService analysis results share the same naming rule.